src/seedu/addressbook/data/person/Person.java

```diff
@@ -84,4 +87,15 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    public String getPrintableString(Printable... printables) {
+        // string initialized with first printable (default)
+        String printableString = printables[0].getPrintableString();
```

Comment:
can you not add this into the loop instead of calling index 0 outside? think why or why not!